web_group_expand/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Group Expand Buttons",
+    "category": "Web",
+    "version": "1.0",
+    "author": "OpenERP SA, "
+               "AvanzOSC, "
+               "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+               "Odoo Community Association (OCA)",
+    "depends": [
+        "web"
+    ],
+    "js": [
+        "static/src/js/web_group_expand.js",
+    ],
+    "qweb": [
+        "static/src/xml/expand_buttons.xml",
+    ],
+    "css": [
```

Comment:
the css key does nothing since 8, same as javascript, infact your styling is not applied (just tried on runbot)